src/ontology/mondo-edit.obo

```diff
@@ -578901,7 +578907,7 @@ xref: MEDGEN:1794276 {source="MONDO:equivalentTo", source="MONDO:MEDGEN"}
 xref: OMIM:619701 {source="MONDO:equivalentTo"}
 xref: UMLS:C5562066 {source="MONDO:equivalentTo", source="MONDO:MEDGEN", source="MEDGEN:1794276"}
 is_a: MONDO:0700092 {source="OMIM:619701"} ! neurodevelopmental disorder
-relationship: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/25590 {source="OMIM:619701"} ! OGDHL
+relationship: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/25590 ! OGDHL
```

Comment:
removal of omim source looks good https://omim.org/entry/619701

@sabrinatoro do you want to remove the entire gene association as well since there is no other source provenance and omim states:
```
The existence of a monogenic disorder caused by mutation in the OGDHL gene ([617513](https://omim.org/entry/617513)), designated Yoon-Bellen neurodevelopmental syndrome, which was reported by [Yoon et al. (2017)](https://omim.org/entry/619701#3) and [Yap et al. (2021)](https://omim.org/entry/619701#2), has been called into question by [Lin et al. (2023)](https://omim.org/entry/619701#1).
```

Reply:
Yes, we should remove this association.  Thank you.

Reply:
Since you are currently editing, can you remove this association?